src/Neo/SmartContract/Native/TransactionState.cs

```diff
@@ -21,6 +21,11 @@ namespace Neo.SmartContract.Native
     /// </summary>
     public class TransactionState : IInteroperable
     {
+        /// <summary>
+        /// The transaction is trimmed.
```

Comment:
Why the state `trimmed` was picked as a name. I did not fully connected it to the conflict yet.
Is there any part actually trimmed or cut from the `tx`?

Reply:
> Is there any part actually trimmed or cut from the tx?

Yes. Trimmed transaction contains only its hash filled. That's why it's "trimmed" (see also https://github.com/neo-project/neo/pull/2818#issuecomment-1499181901). We can choose any other name, if you'd like to.

Reply:
Thanks @AnnaShaleva, I will double check soon.